src/bot/commands/skip.ts

```diff
@@ -0,0 +1,41 @@
+import { CommandInteraction } from 'discord.js';
+import BotContext from '../bot-context';
+import Command from './command';
+
+const skipCommandOption = 'count';
+export class SkipCommand extends Command {
+  constructor() {
+    super('skip', 'Skip currently playing sound(s)');
+    this.commandData.options = [{
+      name: skipCommandOption,
+      type: 'STRING',
+      required: false,
+      description: 'Enter amount to skip (including currently playing) or "all" to clear the queue entirely.',
+    }];
+  }
+
+  execute(interaction: CommandInteraction, context: BotContext): Promise<any> {
+    const skipOption = interaction.options.getString(skipCommandOption, false);
```

Comment:
```suggestion
    const skipOption = interaction.options.getString(skipCommandOption);
```

The second parameter is optional and defaults to false so we might aswell omit it.